configuration_zsh/.vimrc

```diff
@@ -1,15 +1,51 @@
-syntax on
-set number
-set showbreak=+++
-set linebreak
-set showmatch
-set hlsearch
-set autoindent
-set tabstop=4
-set shiftwidth=4
-set smartindent
-set smarttab
-set softtabstop=4
-set ruler
-set backspace=indent,eol,start
-set expandtab
+"=============Colors============= 
+colorscheme default		" colorscheme if you so wish
```

Comment:
@NOhs did you have a colorscheme somewhere, which you wanted to use? Or is the default fine with our colors palette?